backends-velox/src/test/scala/org/apache/gluten/execution/ScalarFunctionsValidateSuite.scala

```diff
@@ -664,6 +665,15 @@ class ScalarFunctionsValidateSuite extends FunctionsValidateTest {
     }
   }
 
+  test("Test sequence function") {
+    withSQLConf(("spark.sql.optimizer.excludedRules", NullPropagation.ruleName)) {
```

Comment:
Curious why this rule matters, instead of constant folding rule. Could you clarify?

Reply:
hey good question @PHILO-HE , actually the matter is `constant folding rule` which should be removed from the `excludedRules`, so here is only use to override and remove it from default test sql conf https://github.com/apache/incubator-gluten/blob/main/backends-velox/src/test/scala/org/apache/gluten/execution/FunctionsValidateTest.scala#L44

Reply:
With constant folding rule applied, will the physical plan delivered to Gluten become an `Alias` plan (the result is actually computed by Spark, and Gluten just offloads the `Alias` plan)? Could you please verify?

Reply:
hey @PHILO-HE, maybe you mis-understand here,  I just removed `constant folding rule` from `excludedRules` to make sure the rule can take effect (the default spark behavior), so the result of the UT is computed actually by spark and been put into `unsafeArray` with Alias Literal type to offload to gluten.

if put `constant folding rule` in `excludedRules`, it will try offload `sequence` to velox and fallback due to unsupporetd function name.

Reply:
@gaoyangxiaozhu, I see. I think it would be better to revise test name to the one like: "sequence function optimized by Spark constant folding"